Korrektur lesen, Kommentare s. Discord-Thread

Sirtario opened the pull request.

RonjaDr committed:
Korrektur lesen, Kommentare s. Discord-Thread

reichts commented on Main.tex:
```diff
@@ -15,50 +15,19 @@
- In dieser Arbeit soll die Frage geklärt werden welche Methodiken möglich sind bzw. angewandt werden, um die Erzeugung ethisch 
- fragwürdiger Inhalte durch IGAI zu verhindern?
+
+Die Entwicklung von Kunst mithilfe von KI reicht zurück bis in die 1960er Jahre\cite{Garcia}, aber erst in den letzten Jahren wurden diese KIs durch die Veröffentlichungen von Projekten wie DALL-E (01/21), Midjourney (03/22) oder Stable Diffusion (08/22) der Allgemeinheit zugänglich gemacht. Auslotungen der Leistungsfähigkeit dieser KIs zeigen jedoch auf, dass sie sexistische oder rassistische Ve ...
```
"in den letzten Jahren" hatte Herrn Müller ja nicht geschmeckt, ist das ok dadurch, dass die Monate und Jahre bei den Sachen dahinter stehen, oder lieber "in den letzten zwei Jahren"?

MichelHecker replied:
würde einfach den zeitraum von chatgpt veröffentlichung und heute nehmen oder so. Auf jeden Fall einen konkreten Zeitraum

reichts commented on Main.tex:
```diff
@@ -15,50 +15,19 @@
- fragwürdiger Inhalte durch IGAI zu verhindern?
+
+Die Entwicklung von Kunst mithilfe von KI reicht zurück bis in die 1960er Jahre\cite{Garcia}, aber erst in den letzten Jahren wurden diese KIs durch die Veröffentlichungen von Projekten wie DALL-E (01/21), Midjourney (03/22) oder Stable Diffusion (08/22) der Allgemeinheit zugänglich gemacht. Auslotungen der Leistungsfähigkeit dieser KIs zeigen jedoch auf, dass sie sexistische oder rassistische Ve ...
+Angesichts dessen haben u. a. die Entwickler von Midjourney reagiert und das Verwenden von Wörtern mit Bezug zur Thematik menschlicher Fortpflanzungssysteme verboten \cite{Heikkilae} und ihre kostenlose Testversion eingestellt. \cite{NelsonMidjourney} Daraus ergibt sich die Frage, welche Methodiken möglich sind bzw. angewandt werden, um die Erzeugung ethisch fragwürdiger Inhalte durch IGAI zu ver ...
```
Müsste die zweite Citation nicht vor den Punkt?

reichts commented on Main.tex:
```diff
@@ -82,48 +52,46 @@ \subsection{Ethik}\label{subsection:ethicsdefinition}
-Im Gegensatz zu bisherigen AIs können den generative artifical intelligence (GAI) neue Daten generieren, anstatt wie bisher Daten zu analysieren oder basierend auf gegebenen Daten zu handeln. Außerdem werden ihnen zum Trainieren enorme Datenmengen, 
-wie der komplette Inhalt von Wikipedia, zur Verfügung gestellt. Daher sind diese AIs erst mit steigender Rechenkapazität möglich geworden. GAIs modellieren Netze aus miteinander verbundenen Daten in verschiedenen multimedialen Formaten. Demnach 
-bestünden beispielsweise Verbindungen zwischen dem Wort Panda, ein Bild von einem Panda und einem Video von einem Panda. Dadurch können GAIs ein beliebiges Eingabeformat in ein beliebiges Ausgabeformat übersetzen. Thema dieser Arbeit sind image generating 
+Im Gegensatz zu bisherigen AIs können generative artifical intelligence (GAI) neue Daten generieren, anstatt wie bisher Daten zu analysieren oder basierend auf gegebenen Daten zu handeln. Außerdem werden ihnen zum Trainieren enorme Datenmengen, wie der komplette Inhalt von Wikipedia, zur Verfügung gestellt. Diese AIs sind daher erst mit steigender Rechenkapazität möglich geworden. GAIs modelliere ...
```
Hier müsste Michel dann noch ne Citation zu bringen? also wikipedia und rechenkapazität

reichts commented on Main.tex:
```diff
@@ -82,48 +52,46 @@ \subsection{Ethik}\label{subsection:ethicsdefinition}
+
 \section{Nicht-Technische Methoden}
-Nicht-Technische Methoden sind solche, die nicht primär auf Technische Lösungen setzen.
+nicht-technische Methoden sind solche, die nicht primär auf technische Lösungen setzen.
```
was ein Humbug, das war mein Vorschlag dazu: "Nicht-technische Ansätze sind hierbei solche, die nicht direkt in die Funktionsweise der IGAI eingreifen. Diese Methoden liegen damit nicht in der Verantwortung des Entwicklers der IGAI, sondern in der Verantwortung des Anbieters oder des Anwenders der IGAI."

reichts replied:
wobei der zweite Satz da eigentlich auch so nicht mehr rein gehört

MichelHecker replied:
den Vorschlag finde ich soweit gut, bis auf den zweiten Satz.

Sirtario replied:
Naja wenn mit dem Entwickler nur der Hans am Laptop gemeint ist, der den Code schreibt passt das...

reichts committed:
Änderungen von Herrn Müller und Franke bei meinem
Zeug umgesetzt

RonjaDr committed:
Korrektur technischer und nicht-technischer Methoden sowie Kommentare auf Discord

RonjaDr committed:
Fazit

RonjaDr committed:
Merge branch 'RonjasBranch' of https://github.com/Sirtario/Inf-Gesell into RonjasBranch

# Conflicts:
#	Main.tex

reichts committed:
merge gefixed
Verzerrung haben Zitationen bekommen
überwachtes lernen hat quelle bekommen,
quellen der lernmethoden muessen noch eingearbeitet werden

reichts committed:
michels Änderungen händisch eingefügt
brnach michel_anpassungen ist damit obsolet

reichts committed:
Text bei Lernmethoden angepasst,
Lewrnmethoden Quellen hinzugefuegt
Text umstrukturiert, Nummerierung gefixt

reichts committed:
kleinere Anpassungen

reichts committed:
Khaldons Änderungen hinzugefügt

reichts committed:
kleine Änderung Zitationen Einleitung

Commit:
nontechnical important citation

MichelHecker committed:
fixed 10 Main.tex

reichts committed:
Aenderungswünsche von Ronja eingearbeitet
Aenderungen von Michel ergänzt
Aenderungen von Khaldon ergänzt

Sirtario committed:
Add Acronym index

Sirtario committed:
Add links

Sirtario committed:
Merge pull request #7 from Sirtario/Abkürzungsverzeichniss-erstellen

Add Acronym index

Commit:
-abstract
-label gefixed
-quellen gefixed

Sirtario committed:
Merge branch 'main' into RonjasBranch
